antenna/throttler.py

```diff
@@ -178,6 +178,31 @@ def always_match(crash):
     return True
 
 
+def match_infobar_true(data):
+    """Matches crashes we need to filter out due to infobar bug
+
+    Bug #1425949.
+
+    """
+    product = data.get('ProductName', '')
+    infobar = data.get('SubmittedFromInfobar', '')
+    version = data.get('Version', '')
+    buildid = data.get('BuildID', '')
+
+    if not (product and infobar and version and buildid):
+        return False
+
+    return (
+        product == 'Firefox' and
+        infobar == 'true' and
+        version.startswith(('52.', '53.', '54.', '55.', '56.', '57.', '58.', '59.')) and
+        # FIXME(willkg): buildid is a string that starts with a YYYYMMDD date
+        # so we're doing string compares. Change this to the buildid date when
+        # we push out fixes.
+        buildid < '20180110'
```

Comment:
consider putting this a bit further out. 59 will go live 2018-03-13

Reply:
just in case

Reply:
I'll push it further out.